portal-impl/src/com/liferay/portal/servlet/ComboServlet.java

```diff
@@ -227,6 +228,8 @@ protected void doService(
 							HttpHeaders.CACHE_CONTROL_NO_CACHE_VALUE);
 						response.setStatus(HttpServletResponse.SC_NOT_FOUND);
 
+						PortalUtil.sendError(HttpServletResponse.SC_NOT_FOUND,new NoSuchAddressException(),request,response);
+
```

Comment:
Don't have to handle the error here. 
Also, NoSuchAddressException is for User/Account address exception. Not for the URL exception.

Reply:
Yes, I will remove it and can you tell me why we don't have to handle the error here, thank you.